package.json

```diff
@@ -41,8 +41,9 @@
     "cli:debug": "node -r ts-node/register -r tsconfig-paths/register --inspect-brk src/cli/index.ts",
     "generate-assets": "node -r ts-node/register ./scripts/generate-assets.ts",
     "inline-version": "./scripts/inline-version.js",
+    "prelint": "tsc --noEmit",
     "lint.fix": "yarn lint --fix",
-    "lint": "tsc --noEmit && tslint 'src/**/*.ts'",
```

Comment:
@P0lip Do we really need to drop that typechecking step?

Reply:
It's moved to prelint. Prelint is always executed before lint, so we don't lose anything.
Type checking is indeed necessary.

Reply:
Perfect! I overlooked that. Thanks!